### PR TITLE
Redesign WP plugin

### PR DESCRIPTION
**Pull request by alexbassy**
We can now add forms without creating an account! ¡Woop!

### CODE REVIEW

File: src/js/index.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+  }
+
+  if (tinyMCE.activeEditor) {
+    tinyMCE.activeEditor.windowManager.open(windowConfiguration(editor, values))
+  } else {
+    tinyMCE.windowManager.open(windowConfiguration(editor, values))
+  }
+}
+
+jQuery(function ($) {
```
Comment (jepser) [inline src/js/index.js:103]: I think is possible to get rid of jQuery here, what do you think?

File: src/sass/_banner.scss [hunk 2]
```diff
@@ -0,0 +1,63 @@
+.tf-banner {
```
Comment (jepser) [inline src/sass/_banner.scss:1]: 👍

File: src/sass/main.scss [hunk 3]
```diff
@@ -0,0 +1,216 @@
+@import 'mixins';
+@import 'editor-button';
+
+$active: #0084bc;
```
Comment (jepser) [inline src/sass/main.scss]: can we put a more descriptive name here?

File: typeform-actions.php [hunk 4]
```diff
@@ -1,26 +1,38 @@
...
+    $typeformObject = array(
+        'pluginRoot'    => tf_plugin_url(),
+        'userEmail'     => $current_user->user_email,
+    );
+
+    wp_register_script( 'tf_tinymce', tf_plugin_url() . 'assets/js/typeform-tinymce.js', [] );
+    wp_localize_script('tf_tinymce', 'typeformObject', $typeformObject);
+    wp_enqueue_script('tf_tinymce');
+
+    // wp_enqueue_script( 'tf_tinymce', tf_plugin_url() . 'assets/js/typeform-tinymce.js', [] );
```
Comment (jepser) [inline typeform-actions.php]: remove unused lines

File: typeform-actions.php [hunk 5]
```diff
@@ -1,26 +1,38 @@
...
+
+    wp_enqueue_style('tf_css', plugin_dir_url( __FILE__ ) . 'assets/css/main.css' );
```
Comment (jepser) [inline typeform-actions.php]: we can use tf_plugin_url() instead of plugin_dir_url( __FILE__ )

File: typeform-actions.php [hunk 6]
```diff
@@ -58,3 +70,8 @@ function tf_add_query_url($url){
     return (isset($_GET) && !empty($_GET)) ? $url . '?' . http_build_query($_GET) : $url;
```
Comment (jepser) [inline typeform-actions.php:67]: did you find a way to exclude common queries?

**Review by jepser — COMMENTED** [review]
Also please check in readme that it says user at least 4.2 WordPress version (or the tinymce that works)

**Comment by jepser** [comment]
@alexbassy I think is this:

readme.txt -> Requires at least: 4.2 change this to **4.4**

File: typeform-actions.php (hunk 6 above)
Comment (alexbassy) [reply typeform-actions.php:67]: No not yet, I think we can do this for the next version

**Comment by alexbassy** [comment]
@jepser I've made the requested changes (minus the common query argument exclusion)

**Comment by jepser** [comment]
👍

**Pull request merged**